README.rst

```diff
@@ -361,6 +362,42 @@ Note that ``THREADS`` backend is not recommended since `Kokkos does not guarante
 
 .. installation_LKokkos-end-inclusion-marker-do-not-remove
 
+.. installation_LTensor-start-inclusion-marker-do-not-remove
+
+Lightning-Tensor installation
+*****************************
+Lightning-Tensor requires CUDA 12 and the `cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`_ (only the `cutensornet <https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html>`_ library is required).
+The SDK may be installed within the Python environment ``site-packages`` directory using ``pip`` or ``conda`` or the SDK library path appended to the ``LD_LIBRARY_PATH`` environment variable.
```

Comment:
I am not sure this line is more helping than confusing, should we just leave it with the link to the SDK installation guide below? (I actually dont know what is best and leave it up to you)